lib/git/pr/release/util.rb

```diff
@@ -94,12 +94,13 @@ def build_pr_title_and_body(release_pr, merged_prs, changed_files, template_path
                        DEFAULT_PR_TEMPLATE
                      end
 
-          erb = if RUBY_VERSION >= '2.6'
-                  ERB.new template, trim_mode: '-'
-                else
-                  ERB.new template, nil, '-'
-                end
-          content = erb.result binding
+          erb = ERB.new template, trim_mode: '-'
+          content = erb.result_with_hash(
+            release_pull_request: release_pull_request,
+            target_pull_request: target_pull_request,
+            merged_pull_requests: merged_pull_requests,
+            pull_requests: pull_requests
+          )
```

Comment:
It's awesome to change `ERB#result(binding)` to `ERB#result_with_hash` !

Also bind other local variables.
I know that `changed_files` is often used in ERB templates.